oak_sign/src/lib.rs

```diff
@@ -81,7 +96,7 @@ pub struct SignatureBundle {
 
 impl SignatureBundle {
     /// Signs a SHA-256 hash of the `input` using `private_key`.
-    pub fn create(input: &[u8], key_pair: &KeyBundle) -> anyhow::Result<SignatureBundle> {
+    pub fn create(input: &[u8], key_pair: &KeyPair) -> anyhow::Result<SignatureBundle> {
```

Comment:
Personally I would just expose `sign` and `verify` as top-level methods in this module, *not* on `SignatureBundle` itself. There is really not much about `SignatureBundle` doing the work here, I think it would read better as `oak_sign::sign` and `oak_sign::verify`.